core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java

```diff
@@ -245,35 +256,159 @@ public void resolveGumCollision(Obstacle bodyA, Obstacle bodyB){
 
         GumModel gum = null;
         Obstacle body = null;
+        EnemyModel enemy = null;
+        TileModel tile = null;
         if (isGumObstacle(bodyA)) {
             gum = (GumModel) bodyA;
             body = bodyB;
+            if (bodyB instanceof EnemyModel) {
+                enemy = (EnemyModel) bodyB;
+            }
         };
         if (isGumObstacle(bodyB)) {
             gum = (GumModel) bodyB;
             body = bodyA;
+            if (bodyA instanceof EnemyModel) {
+                enemy = (EnemyModel) bodyA;
+            }
         };
         if (gum != null && gum.getName().equals("gumProjectile")) {
             // Do this once gum is turning from a projectile to sticky
             gum.setVX(0);
             gum.setVY(0);
             gum.setTexture(bubblegumController.getStuckGumTexture());
             gum.setName("stickyGum");
-            gum.setRadius(gum.getRadius() * 1.5f);
+            gum.setRadius(gum.getRadius());
             // Changing radius resets filter for some reason
             gum.getFilterData().maskBits = MASK_GUM;
```

Comment:
Do we still need to reset filter data if we are no longer changing the radius

Reply:
mm yea I can clean this up

Reply:
pretty sure these are still required after messing w it